app/plugin.go

```diff
@@ -502,6 +502,10 @@ func (a *App) GetMarketplacePlugins(filter *model.MarketplacePluginFilter) ([]*m
 
 		result = append(result, &model.MarketplacePlugin{
 			BaseMarketplacePlugin: &model.BaseMarketplacePlugin{
+				Labels: []model.MarketplaceLabel{{
+					Name:        "Local",
+					Description: "This plugin is not listed in the marketplace but was installed manually",
```

Comment:
should this string be i18n'd? not sure if we have anything that does i18n currently.

Reply:
No, it should not be localized, because the other labels are also not localized. I've filled https://mattermost.atlassian.net/browse/MM-21710 to improve this in the future.